about.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="styles/styles.css">
+    <title>Sai Samant</title>
+  </head>
+  <body>
+    <header>
+      <div class="hello">
+        <h1>&lt; about &gt;</h1>
+      </div>
+
+      <div class="nav">
+        <nav>
+          <ul>
+            <li><a href="index.html">home</a></li>
+            <li><a href="portfolio.html">code</a></li>
+            <li><a href="blog.html">other</a></li>
+
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main >
+      <section class = "about">
+        <p>I am student at Ada Developers Academy where I am training to become a software engineer. We are learning Ruby, Rails, HTM & CSS, JavaScript, test-driven development, and agile and pair-programming practices. A major part of why Ada appeals to me is its <a href="http://adadevelopersacademy.org/program">social-justice mission</a> to help women and people of non-binary gender succeed in tech.</p>
+        <p>Coding and parenting take up most of my time these days, but I have been known to enjoy running on trails in Seattle's many beautiful parks, reading, watching sitcoms, following politics, knitting, eating french fries, and catching up on sleep.</p>
+        <p>I maintain a very spotty social media presence across too many platforms:</p>
+        <ul class = "social-media">
+          <li><a href="https://twitter.com/saisamant", target = "_blank"><img src="assets/twitter.svg" alt="twitter" height="40px"></a></li>
+          <li><a id="ravelry" href="http://www.ravelry.com/people/samtheant", target = "_blank"><img src="assets/ravelry.svg" alt="knit" height="40px"></a></li>
```

Comment:
I feel like your `alt` attributes could be more descriptive here. Remember that the main purpose of an `alt` description is to accurately describe what the image is, both for screen readers and in case the image fails to load. Does "knit" accurately describe this image, or would "ravelry logo" suit better? Similarly for "insta" and "fb", as well as "git" on your code page.